rails-commenting.rb

```diff
@@ -48,15 +48,15 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
-      redirect_to blog_posts_path
+      # ---8)   takes the post (database entry) matching the given unique id and throws it in a volcano.
+      redirect_to blog_posts_path   #                   ***I could use some extra clarification on what's up with blog_posts_path. My understanding is that it's essentially leading back to the 'index', and is a name automatically generated when the controller is first created, but I feel like i'm missing something here...***
```

Comment:
This is saying if a post is destroyed, redirect to the index where we will be able to see that it has been indeed thrown into a volcano.